.github/workflows/compaund-java-sonatype-push.yml

```diff
@@ -92,7 +92,7 @@ jobs:
           ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.sonatypeSigningPassword }}
       - name: Publish Test Results
         uses: EnricoMi/publish-unit-test-result-action@v2
-        if: always()
+        if: always() && github.actor != 'dependabot[bot]'
```

Comment:
note: it is not advised to use `always()`. It is better to use `!cancelled()` instead. To avoid adding brackets you can swap the conditions